src/App.tsx

```diff
@@ -1,61 +1,137 @@
 import './App.scss';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { findUserById, getNewId } from './helpers';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { InitialTodo, Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const users = usersFromServer;
```

Comment:
`users` is the same reference as `usersFromServer` so we can use directly `usersFromServer`